### ISSUE TITLE
[Adaptive Cards-Save/open] : No shortcut keys are defined for the Save/Open/settings/Add new card controls

### ISSUE THREAD

**Issue by PERUAMLLABALAKRISHNA** [microsoft/AdaptiveCards#8043]
### Target Platforms

UWP

### SDK Version

3.0.2210.26001

### Application Name

Adaptive cards

### Problem Description

**Test Environment:**
OS version: 22H2 (Build 25182.1010)
Application: AdaptiveCardVisualizer
Application Version: 3.0.2210.26001
Screen Reader: Narrator

**Repro Steps:**
1.Open the Adaptive Card Visualizer application.
2.Navigate to the tab items in the header region.
3. Observe the issue.

**Actual Result:**
Users are observed that no shortcut keys are defined for the Save/Open/settings/Add new card controls.

**Expected Result:**
For better user experience, shortcut keys should be provided for the Save/Open/settings/Add new card controls.
Example: Ctrl+S for save etc.

**User Impact:**
Users need to navigate the controls manually to perform the task, if no shortcuts are defined.

### Screenshots

<img width="953" alt="No shortcut keys are defined" src="https://user-images.githubusercontent.com/117892125/201130891-b36345ff-303c-44cf-93be-ca01531c6e7a.png">

### Card JSON

```JSON
NA
```

**Comment by vaishali1397** [comment]
Issue has been fixed in below test environment as For New card (ctrl+ N),to open file (Ctrl+O), to save file(Ctrl+ S) and to open settings(Ctrl+,) hence Adding A11yTTValidated tag.
**Test Environment:**
OS Version: 22H2 ( Build 25330.1000)
Applicvation: AdaptiveCardVisualizer
